AFQ/utils/conversion.py

```diff
@@ -0,0 +1,88 @@
+import numpy as np
+
+from tqdm import tqdm
+import scipy.io
+
+from dipy.io.stateful_tractogram import StatefulTractogram, Space
+import nibabel as nib
+
+from AFQ.data import BUNDLE_MAT_2_PYTHON
+
+
+class MatlabFileTracking():
+    """
+    Helper class.
+    Acts the same as a tracking class from DIPY,
+    in that it yields a streamline for each call to __iter__.
+    Initialized with an opened h5py matlab file
+    and the location of the streamlines in that h5py file.
+    """
+
+    def __init__(self, fg_ref):
+        self.fg_ref = fg_ref
+
+    def __iter__(self):
+        for i in tqdm(range(self.fg_ref.shape[0])):
+            yield np.transpose(self.fg_ref[i, 0])
+
+
+def matlab_tractography(mat_file, img):
+    """
+    Converts a matlab tractography file to a stateful tractogram.
+
+    Parameters
+    ----------
+    mat_file : str
+        Path to a matlab tractography file.
+    img : Nifti1Image or str
+        Path to an img file to be loaded with nibabel or an img
+        to serve as the reference for the stateful tractogram.
+
+    Returns
+    -------
+    Dipy StatefulTractogram in RASMM space.
+    """
+    mat_file = scipy.io.loadmat(mat_file)
```

Comment:
I think that if you call `loadmat` with `squeeze_me=True` you con't have to index into singleton dimensions below.